brewtils/rest/client.py

```diff
@@ -618,18 +618,20 @@ def post_jobs(self, payload):
         """
         return self.session.post(self.job_url, data=payload, headers=self.JSON_HEADERS)
 
-    def post_execute_job(self, job_id):
+    def post_execute_job(self, job_id, reset_interval=False):
         # type: (str) -> Response
         """Performs a POST on the Job Execute URL
 
         Args:
             job_id: The ID of the Job
+            reset_interval: Sets interval of job to restart now
 
         Returns:
             Request Response object
         """
+        url_params = "?reset_interval=True" if reset_interval else ""
```

Comment:
This will work, but I'm not sure it's how you wan to do this.  I think you want to pass this info in as a `params` argument to the `post` call.  I'm not 100% positive on the form, so make sure you try this out, but it should be something like:

```python
url_params = {}
if reset_interval:
    urls_params["reset_interval"] = "True"

return self.session.post(
    self.job_url + job_id + "/execute", headers=self.JSON_HEADERS, params=url_params
)
```

While it may all seem the same in this case, the main reason to build it this way is that it's way more manageable as soon as you have more than one query parameter.  You don't want to be in the business of building a properly formatting string for the query parameters.

Reply:
According to the [Requests module docs](https://docs.python-requests.org/en/master/user/quickstart/#passing-parameters-in-urls), this is the correct implementation.